### PR TITLE
RDKTV-11877: timezone interface

### PR DESCRIPTION
**Pull request by npoltorapavlo**
Reason for change: ITimeZone interface for
get/set timezone (ex. in a file).
Test Procedure: Builds.
Risks: Low
Signed-off-by: Nikita Poltorapavlo <npoltorapavlo@productengine.com>

### CODE REVIEW

File: interfaces/ITimeZone.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  struct EXTERNAL INotification : virtual public Core::IUnknown {
+    enum { ID = ID_TIMEZONE_NOTIFICATION };
+
+    virtual void TimeZoneChanged(const string &timeZone) = 0;
+  };
+
+  virtual void Register(ITimeZone::INotification* sink) = 0;
+  virtual void Unregister(ITimeZone::INotification* sink) = 0;
+
+  virtual string GetTimeZone() = 0;
```
Comment (pwielders) [inline interfaces/ITimeZone.h:39]: I guess this can be a const method.

**Review by mhughesacn — CHANGES_REQUESTED** [review]
Sorry Nikita - pilot error here, never pressed <submit>

**Review by pwielders — APPROVED** [review]

**Review by mhughesacn — DISMISSED** [review]

**Comment by mhughesacn** [comment]
Thanks Nikita

**Comment by anand-ky** [comment]
@pwielders - Pls help merge this PR.

**Comment by anand-ky** [comment]
@npoltorapavlo - Pls bring this to R2 branch.

**Comment by npoltorapavlo** [comment]
#128

**Comment by anand-ky** [comment]
@mhughesacn - Confirming that we are good with these changes and can be brought to R2 branch.

**Review by pwielders — DISMISSED** [review]

**Review by pwielders — APPROVED** [review]

**Pull request merged**